pkgs/tools/misc/grex/default.nix

```diff
@@ -28,6 +28,6 @@ rustPlatform.buildRustPackage rec {
     description = "A command-line tool for generating regular expressions from user-provided test cases";
     homepage = "https://github.com/pemistahl/grex";
     license = licenses.asl20;
-    maintainers = with maintainers; [ SuperSandro2000 ];
+    maintainers = with maintainers; [ SuperSandro2000 mfrw ];
   };
```

Comment:
Please include `mainProgram` and `changelog`

Reply:
Sure :)